apps/builder/src/locales/pt.ts

```diff
@@ -21,4 +25,142 @@ export default defineLocale({
+  'billing.invoices.subtotal': 'Subtotal',
+  'billing.preCheckoutModal.companyInput.label': 'Nome da empresa:',
+  'billing.preCheckoutModal.emailInput.label': 'E-mail:',
+  'billing.preCheckoutModal.taxId.label': 'Identificação fiscal:',
+  'billing.preCheckoutModal.taxId.placeholder': 'Tipo de ID',
+  'billing.preCheckoutModal.submitButton.label':
+    'Ir para a finalização da compra',
+  'billing.pricingCard.heading': 'Mudar para {plan}',
+  'billing.pricingCard.perMonth': '/ mês',
+  'billing.pricingCard.plus': ', mais:',
+  'billing.pricingCard.upgradeButton.current': 'Sua assinatura atual',
+  'billing.pricingCard.chatsPerMonth': 'chats/mês',
+  'billing.pricingCard.chatsTooltip':
+    'Um chat é contado sempre que um usuário inicia uma discussão. Ele é independente do número de mensagens que ele envia e recebe.',
+  'billing.pricingCard.storageLimit': 'GB de armazenamento',
+  'billing.pricingCard.storageLimitTooltip':
+    'Você acumula armazenamento para cada arquivo que seu usuário carrega em seu bot. Se você excluir o resultado, ele liberará espaço.',
+  'billing.pricingCard.starter.description':
+    'Para indivíduos e pequenas empresas.',
+  'billing.pricingCard.starter.includedSeats': '2 assentos incluídos',
+  'billing.pricingCard.starter.brandingRemoved': 'Marca removida',
+  'billing.pricingCard.starter.fileUploadBlock': 'Bloco de entrada de arquivo',
+  'billing.pricingCard.starter.createFolders': 'Criar pastas',
+  'billing.pricingCard.pro.mostPopularLabel': 'Mais popular',
+  'billing.pricingCard.pro.description':
+    'Para agências e startups em crescimento.',
+  'billing.pricingCard.pro.everythingFromStarter': 'Tudo em Starter',
+  'billing.pricingCard.pro.includedSeats': '5 assentos incluídos',
+  'billing.pricingCard.pro.customDomains': 'Domínios personalizados',
+  'billing.pricingCard.pro.analytics': 'Análises aprofundadas',
+  'billing.usage.heading': 'Uso',
+  'billing.usage.chats.heading': 'Chats',
+  'billing.usage.chats.alert.soonReach':
+    'Seus typebots são populares! Você logo alcançará o limite de chats de seu plano. 🚀',
+  'billing.usage.chats.alert.updatePlan':
+    'Certifique-se de atualizar seu plano para aumentar esse limite e continuar conversando com seus usuários.',
+  'billing.usage.chats.resetInfo': '(reiniciado todo dia 1)',
+  'billing.usage.storage.heading': 'Armazenamento',
+  'billing.usage.storage.alert.soonReach':
+    'Seus typebots são populares! Você logo alcançará o limite de armazenamento de seu plano. 🚀',
+  'billing.usage.storage.alert.updatePlan':
+    'Certifique-se de atualizar seu plano para continuar coletando arquivos enviados. Você também pode excluir arquivos para liberar espaço.',
+  'billing.limitMessage.brand': 'remover a marca',
+  'billing.limitMessage.customDomain': 'adicionar domínios personalizados',
+  'billing.limitMessage.analytics': 'desbloquear análises aprofundadas',
+  'billing.limitMessage.fileInput': 'usar blocos de entrada de arquivo',
```

Comment:
`usar blocos de entrada de arquivo` should be replaced by `usar blocos de envio de arquivo`